canonical/ubuntu.com
# Limit Beta Attempts

## Done

- A candidate can schedule only a limited number of exams for the beta (1 exam per candidate for now)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Login and go to /credentials/schedule
    - If you donot have an exam scheduled, you should be able to schedule an exam
    - If you have an exam scheduled, going to /credentials/schedule should give an error
    - You should be able to reschedule the scheduled exam
    - If you cancel the scheduled exam, you should be able to schedule another one without problems.

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
